scapy/ansmachine.py

```diff
@@ -11,6 +11,8 @@
 #  Answering machines  #
 ########################
 
+from __future__ import annotations
```

Comment:
We no longer support Python 2, so this is not wanted.

Reply:
Thanks for your review!

This is still necessary if you want to use modern type hints such as `dict[str]` instead of `Dict[str]` https://peps.python.org/pep-0585/

I can revert to `Dict` if you prefer, it just requires an extra `from typing import Dict`